src/Controls/src/Core/DependencyService.cs

```diff
@@ -90,6 +90,14 @@ public static class DependencyService
 		static Type FindImplementor(Type target) =>
 			DependencyTypes.FirstOrDefault(t => target.IsAssignableFrom(t));
 
+		// Once we get essentials/cg converted to using startup.cs
+		// we will delete the initialize code from here and just use
+		// explicit assembly registration via startup code
+		internal static void SetToInitialized()
```

Comment:
Maybe use the `Initialize(Assembly[] assemblies)` and pass an empty array? Then we can keep this API semi-sane for now. Especially since this might have to live on.

Reply:
My thinking here is that all the Initialize code inside `Forms.Init` will just go away. Once this is all rounded out there is no reason to call `Initialize` on the static `Forms` class

If users want to scan assemblies for renderers then they will just call some extension method that we add
https://github.com/dotnet/maui/issues/1156